### ISSUE TITLE
babel rule not found

### ISSUE THREAD

**Issue by demiurg** [tu4mo/gatsby-plugin-babel-exclude#1]
It looks like rules changed in latest gatsby. I'm getting:
  Error: gatsby-plugin-babel-exclude: Could not find rule for /\.(js|mjs|jsx)$/ from webpack config

Possible rules in  gatsby@4.11.1 are:
```
rules [
  /\.mjs$/i,
  /\.js$/i,
  /\.(js|mjs|jsx|ts|tsx)$/,
 ...
  /\.tsx?$/
]
```

**Comment by tu4mo** [comment]
Can you make a PR?

**Comment by demiurg** [comment]
I would, but I am not sure which one is the "right" rule

**Comment by tu4mo** [comment]
Fixed in v2.0.0.
